feat: add recipe for Indian Style pizza (#38)
open-sauced/pizza-verse: pizza-recipes.md

6. Cook for 30 mins, adjust water if thick.

7. Let it cool. Your pizza sauce is ready to use.

> # Chicken Tikka Masala Pizza ( India )

Reviewer: [suggestion] Since you're describing a recipe. You don't need to mention that it's from India. 
```suggestion
# Chicken Tikka Masala Pizza 
```

Author: @CBID2 I have done changes for my end and please do not forget to add the hactoberfest tag😀

3. While the chicken is cooking, prepare the Tomato Sauce and Spice Mix in separate bowls. Remove the chicken from the pan when it is done and discard the juice.

4. Heat a small sauce pan and melt the butter. Sauté the garlic and jalapeno until aromatic. Pour in the Spice Mix and sauté until fragrant, about 1-2 minutes on medium heat. Remove the jalapeno and discard.

5. Add in the Tomato Sauce and stir well to mix. Sprinkle salt to taste and continue to simmer until the sauce thickens, another 2 minutes. Turn off the heat and add the chicken meat into the sauce and mix well.

6. Pre-heat the oven to 450°F (232°C). Place the pizza crust on a pizza pan and brush lightly with olive oil. Spread the sauce/gravy on the pizza crust. Sprinkle 1/2 cup of cheese. Arrange the chicken on the pizza. Top it with the sliced onions, chopped cilantro and the remaining cheese.

7.Reduce the temperature to 425°F (218°C). Bake the pizza for 8-10 minutes. Remove the pizza from the pan when it is done and garnish the remaining chopped cilantro.

> - Notes : Bake according to the instructions on your pizza crust. Depending on the type of crusts you buy, the time and temperature may be slightly different. If you prefer, you can stir fry the chicken instead of grilling it.

Reviewer: [question] If a person makes their own pizza crust, how long should they bake it? Consider adding this information here. 
```suggestion
- **Note:** Bake according to the instructions on your store-bought pizza crust. Depending on the type of crusts you buy, the time and temperature may be slightly different. If you prefer, you can stir fry the chicken instead of grilling it.
```

1. While waiting for the Pizza dough to rise a second time, go on and wash the vegetables used for the pizza topping, slice them, and set aside
2. Fry the minced beef in a small quantity of vegetable oil until it turns pale.
3. Get the tomato sauce ready for the pizza. Here we made use of stock cube, onion powder salt, and some Tomato stews i.e. fresh tomato puree fried in vegetable oil. Add the stock cube, onion powder, and salt to the vegetable oil. You can also add any ingredient of your choice.

## Back to the Pizza dough
### Instructions
1. Bring out the puffed dough from the oven and spread it out more to manipulate it again
2. Spread some more olive oil on the dough.
3. Add the tomato sauce.
4. Sprinkle the cheese on top.

## Back to Baking!
### Instructions
1. Place the pizza in a preheated oven (180°C or 360F) and bake for 20-25 minutes or till the pizza crust is light brown and the cheese has turned watery by the heat.
2. The pizza should be removed from the oven and place the rest of the toppings on top.
3. Place it back in the oven and bake for a maximum of 2 minutes; of course, you do not want the vegetables to become dull.

Enjoy with a chilled soft drink.

> # Chicken Tikka Masala Pizza 

Reviewer: [suggestion] Add an image of the pizza here. It'll help readers understand how this pizza should look. 
```suggestion
# Chicken Tikka Masala Pizza 
<!-- img here --> 
```
